.halcyon/cabal-version

```diff
@@ -0,0 +1 @@
+1.22.4.0
```

Comment:
Currently, the GHC version is automatically inferred from the version constraint of the `base` package, in your [`constraints`](https://github.com/purefn/hiplambda/blob/master/.halcyon/constraints#L19) file:
```
base-4.8.0.0
```

This is described in the [“Make a change”](https://halcyon.sh/tutorial/#make-a-change) section of the tutorial.

This is likely to change in the future (https://github.com/mietek/halcyon/issues/52#issuecomment-79626776), and a `ghc-version` magic file will be accepted then.